CONTRIBUTING.md

```diff
@@ -50,17 +50,84 @@ type checking. Any code changes that you contribute should pass these checks.
 The easiest way to get set up for development is to run the following script:
 
 ```bash
+# first fork the repo on github
+
+#then clone your fork
+git clone https://github.com/<your-github-username>/gluon-ts.git
+
+# check if condo is up to date
+Conda update conda
```

Comment:
You can simply suggest working with virtual environments, and link to [this](https://docs.python.org/3/tutorial/venv.html) and potentially something about pyenv and Conda.

Reply:
I understand if you prefer python virtual environments instead of conda, and the vast majority of python developers on github use conda, including for virtual envs. I'll add this link and remove the setup description. These changes are leaving too much effort on the part of the contributor though.